packages/components/src/Avatar/Avatar.module.css

```diff
@@ -0,0 +1,163 @@
+.wrapper {
+  --avatar-sm: 1.25rem;
+  --avatar-md: 3rem;
+  --avatar-lg: 4.5rem;
+  --avatar-xl: 6rem;
+  --avatar-xxl: 7.75rem;
+
+  background: var(--color-gray-300);
+  border-radius: 100%;
+  box-sizing: border-box;
+  overflow: hidden;
+  align-items: center;
+  display: flex;
+  justify-content: center;
+
```

Comment:
What I'm thinking is something like:
```
width: var(--avatar-x-y);
height: var(--avatar-x-y);
```
here and then in the sizes we can just update the value of the CSS variable, ie:
```
  &.small {
    --avatar-x-y: 1.25rem;
  }
```